src/Calendar/Calendar.js

```diff
@@ -678,7 +679,10 @@ class Calendar extends Component {
 
         const calendarClasses = classnames(
             'fd-calendar',
-            className
+            className,
+            {
+                'fd-calendar--compact': compact
+            }
```

Comment:
It's possible className would override a css rule from `rd-calendar--compact` and it looks like the convention in fundamental-react is to keep the custom className last

```js
        const calendarClasses = classnames(
            'fd-calendar',
            {
                'fd-calendar--compact': compact
            },
            className
        );
```

Reply:
Changed.